### ISSUE TITLE
Sunfire doesn't show as guardian druid

### ISSUE THREAD

**Issue by harsjo** [Hekili/hekili#2138]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.

### WoW Version

Retail (Dragonflight)

### Describe the Issue

I've tried including sunfire in my action list as a guardian druid but it doesn't show at all. Working around it by using wrath instead but set up the conditions as if it was sunfire.
Works fine as moonkin spec, but not as guardian spec, regardless of form.

### How to Reproduce

.

### Player Information (Link)

```Text
build: v10.0.2-2.0.4
level: 70 (70)
class: DRUID
spec: guardian

talents: after_the_wildfire
    astral_influence = 2/2
    berserk_persistence = 1/1
    berserk_ravage = 1/1
    brambles = 1/1
    circle_of_life_and_death = 1/1
    convoke_the_spirits = 1/1
    elunes_favored = 1/1
    feline_swiftness = 2/2
    flashing_claws = 2/2
    frenzied_regeneration = 1/1
    front_of_the_pack = 1/1
    fury_of_nature = 2/2
    galactic_guardian = 1/1
    gore = 1/1
    gory_fur = 1/1
    heart_of_the_wild = 1/1
    improved_stampeding_roar = 1/1
    improved_sunfire = 1/1
    innate_resolve = 1/1
    ironfur = 1/1
    killer_instinct = 3/3
    lycaras_teachings = 3/3
    mangle = 1/1
    maul = 1/1
    moonkin_form = 1/1
    nurturing_instinct = 3/3
    rage_of_the_sleeper = 1/1
    rake = 1/1
    rip = 1/1
    scintillating_moonlight = 2/2
    skull_bash = 1/1
    soothe = 1/1
    soul_of_the_forest = 1/1
    stampeding_roar = 1/1
    starfire = 1/1
    starsurge = 1/1
    sunfire = 1/1
    survival_instincts = 1/1
    survival_of_the_fittest = 2/2
    swipe = 1/1
    thick_hide = 2/2
    thrash = 1/1
    tiger_dash = 1/1
    twin_moonfire = 1/1
    typhoon = 1/1
    ursine_adept = 1/1
    ursocs_guidance = 1/1
    vulnerable_flesh = 2/2

pvptalents: malornes_swiftness
   master_shapeshifter
   sharpened_claws

covenant: none

conduits: none

soulbinds: [niya]

sets: none

gear: bilgewater_cartel_tabard = 1
    bottle_of_spiraling_winds = 1
    circle_of_ascended_frost = 1
    drakebreakers_vest = 1
    drakebreakers_waders = 1
    drakebreakers_wrap = 1
    ferocious_hyena_hidebinders = 1
    goldenwinged_rod = 1
    grals_discarded_tooth = 1
    heart_of_azmerloth = 1
    lavabearer_legwraps = 1
    mantle_of_yearned_freedom = 1
    ottuk_hide_helm = 1
    ottuk_hide_sash = 1
    platinum_star_band = 1
    sightless_mantle = 1
    subjugators_chilling_grips = 1

legendaries: none

itemIDs: 191993, 191995, 192797, 193642, 193651, 193697, 193708, 193731, 193758, 193781, 193793, 195210, 198573, 198579, 198959, 64884, 98093

settings: aoe = 3
    buffPadding = 0
    combatRefresh = 0.1
    custom1Name = Custom 1
    custom2Name = Custom 2
    cycle = false
    cycle_min = 6
    damage = true
    damageDots = false
    damageExpiration = 6
    damageOnScreen = true
    damagePets = false
    damageRange = 0
    debuffPadding = 0
    enabled = true
    enhancedRecheck = false
    gcdSync = true
    maxTime = 10
    nameplateRange = 8
    nameplates = true
    noFeignedCooldown = false
    package = Skotet
    petbased = false
    potion = spectral_agility
    regularRefresh = 0.5
    throttleRefresh = false
    throttleTime = false
    catweave_bear = false
    ironfur_damage_threshold = 15
    mangle_more = false
    maul_rage = 20
    owlweave_bear = false
    shift_for_convoke = true

toggles: cooldowns = true 
    custom1 = false 
    custom2 = false 
    defensives = true 
    essences = true [overridden]
    interrupts = true 
    mode = automatic 
    potions = false 

keybinds: 
    barkskin              = S3  [05]
    berserk               = S4  [05]
    brutal_slash          = 1   [07], 1   [08], 1   [09]
    convoke_the_spirits   = G   [01], G   [07], G   [09], G   [10]
    cyclone               = CF  [06]
    emerald_slumber       = 6   [01], S6  [05]
    entangling_roots      = SF  [06]
    ferocious_bite        = 5   [07]
    frenzied_regeneration = Q   [09]
    growl                 = T   [09]
    heart_of_the_wild     = F3  [06]
    incapacitating_roar   = <   [08], §  [09]
    ironfur               = 3   [09]
    maim                  = E   [07]
    mangle                = 2   [09]
    mark_of_the_wild      = F4  [06]
    maul                  = E   [09]
    moonfire              = 4   [09], 4   [10]
    prowl                 = §  [01], §  [07], Q   [08]
    rage_of_the_sleeper   = S1  [05]
    rake                  = 4   [07]
    rebirth               = SG  [06]
    regrowth              = F2  [06]
    remove_corruption     = SE  [06]
    renewal               = F1  [06]
    rip                   = 3   [07]
    shred                 = 2   [07], 2   [08]
    skull_bash            = <   [07], <   [09]
    stampeding_roar       = S5  [05]
    starfire              = <   [01], 1   [10]
    starsurge             = 1   [01], 5   [10]
    sunfire               = 2   [01], 6   [09], 2   [10], 4   [11]
    survival_instincts    = S2  [05]
    swiftmend             = E   [01]
    swipe                 = 1   [07], 1   [08], 1   [09]
    swipe_bear            = 1   [07], 1   [08], 1   [09]
    swipe_cat             = 1   [07], 1   [08], 1   [09]
    thrash                = E   [02], 6   [07], 5   [09]
    thrash_bear           = E   [02], 6   [07], 5   [09]
    thrash_cat            = E   [02], 6   [07], 5   [09]
    tiger_dash            = T   [07], T   [08]
    travel_form           = X   [05]
    typhoon               = SQ  [06]
    wild_charge           = SSPACE[06]
    wrath                 = 3   [01], 3   [10]

warnings: none
```

### Error Messages (Link)

```Text
no error messages, simply ignores sunfire action.
```

**Comment by Hekili** [comment]
This would not be a bug; this would be a Recommendations ticket.  I'll need you to provide a snapshot from when you thought Sunfire should be recommended but it was not.

A snapshot is required to diagnose this issue.

## Snapshot

Please supply a Snapshot of the addon's decision-making when you are seeing this issue in-game. This is not a screenshot. To generate a Snapshot, please complete the following steps.

 * Log into your WoW character.
 * Recreate the issue you are reporting (i.e., generate 5 Holy Power).
 * When you see the recommendation you disagree with, press ALT-SHIFT-P to Pause and Snapshot (or use ALT-SHIFT-[ to Snapshot without pausing).
   * You can change these keybindings in /hekili > Toggles if needed.
   * If you take the snapshot at a different time, it will not be useful and you'll be asked to submit a new one.
 * Type /hekili and press Enter.
 * Open the Snapshots section on the left side.
 * Select the snapshot that shows the recommendation that you disagree with.
   * Snapshots are labeled with your specialization, the display in question (usually, you'll choose Primary), and the recommendations the addon made with their timings.
 * Click in the Export Snapshot box, press CTRL+A to select all text including hidden text and press CTRL+C to copy.
 * Open [https://pastebin.com](https://github.com/Hekili/hekili/issues/Pastebin), use CTRL+V to paste the snapshot text, then click Create New Paste.
 * Provide the Pastebin ***link*** in a new comment.

This step is essential, as most issues are related to specific classes, specializations, gear, talent choices, or other game systems.  If you do not provide this information, I cannot triage your problem.

**Comment by harsjo** [comment]
I think it's a bug though. I added Sunfire to the top of Default just to try and the snapshot says ability unknown:
https://pastebin.com/ks93kH1e

To clarify: I'm not asking you to add Sunfire to the recommended priority for Guardian. I'm just trying to use it in my custom action list (for pulling mobs) when I noticed it's not working. It is a part of the default owlweave action list though so it probably doesn't work there either.

Thanks for all your hard work!
